cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -176,9 +176,10 @@ data:
   # Labels to add to the CSI RBD Deployments and DaemonSets Pods.
   # ROOK_CSI_RBD_POD_LABELS: "key1=value1,key2=value2"
 
-  # (Optional) Ceph Provisioner NodeAffinity.
+  # (Optional) Ceph CSI provisioner NodeAffinity(applied to both CephFS and RBD provisioner).
```

Comment:
Is this appended to the rbd or cephfs-specific affinity? Or is this just the fallback if those rbd or cephfs affinities aren't specified? This gives the impression that this will also be applied in addition to the separate settings for cephfs or rbd. I think it's fine to keep it separate, let's just clarify the behavior either here or in the docs.

Reply:
Both tolerations and affinites are overridden by specifying rbd/cephfs ones and it also falls back in case of error to common ones. Added the following to be more precise. 
```
  # (Optional) Ceph CSI RBD provisioner NodeAffinity(if specified, overrides CSI_PROVISIONER_NODE_AFFINITY).
  # CSI_RBD_PROVISIONER_NODE_AFFINITY: "role=rbd-node"
  # (Optional) Ceph CSI RBD provisioner tolerations list(if specified, overrides CSI_PROVISIONER_TOLERATIONS).
```